packages/@ember/application/tests/reset_test.js

```diff
@@ -129,9 +128,6 @@ moduleFor(
           let location = initialRouter.get('location');
 
           assert.equal(location.getURL(), '/one');
-          expectDeprecation(() => {
-            assert.equal(get(initialApplicationController, 'currentPath'), 'one');
```

Comment:
Similar to above, can this assertion be changed to assert the router service with `currentRoutePath`? https://api.emberjs.com/ember/3.27/classes/RouterService/properties/currentRouteName?anchor=currentRouteName

This is an integration test, and I wouldn't want to lose the coverage that Ember's routing system in general is up to date after a visit call.